web/index.html

```diff
@@ -70,287 +68,350 @@ <h3 class="sub-header">
-                                                <td class="item" :class="{'item-used': !saveData.includes(recipe.B)}"
-                                                    @click="navigateTo(recipe.B);">
-                                                    {{ data.index[recipe.B][0] }}&nbsp;{{ data.index[recipe.B][1] }}
-                                                </td>
-                                                <td>=</td>
-                                                <td class="item" :class="{'item-used': !saveData.includes(recipe.C)}"
-                                                    @click="navigateTo(recipe.C);">
-                                                    {{ data.index[recipe.C][0] }}&nbsp;{{ data.index[recipe.C][1] }}
-                                                </td>
-                                            </tr>
-                                        </table>
-                                    </transition>
-                                    <div v-else>
-                                        No recipes found
-                                    </div>
-                                    <div v-if="currentHiddenFromLength > 0">
-                                        <div v-if="!showHiddenRecipes" class="text-link"
-                                            @click="showHiddenRecipes = true">
-                                            {{currentHiddenFromLength}} entries hidden
-                                        </div>
-                                        <div v-else class="text-link" @click="showHiddenRecipes = false">
-                                            {{currentHiddenFromLength}} entries shown
+                                    <transition name="slide-right" mode="out-in">
+                                        <div :key="this.elementLoaded">
+                                            <div v-if="!this.elementLoaded"
+                                                style="font-weight: bold; padding-top: 5px;">
+                                                Loading data...
+                                            </div>
+                                            <transition name="slide-right" mode="out-in"
+                                                v-else-if="currentFromLength > 0">
+                                                <div :key="currentElement">
+                                                    <table style="font-weight: bold;">
+                                                        <tr v-for="recipe in currentFrom">
+                                                            <td class="item"
+                                                                :class="{'item-used': !saveData.has(recipe.A)}"
+                                                                @click="navigateTo(recipe.A);"
+                                                                @mouseover="loadChunk(getChunk(recipe.A))">
```

Comment:
Added a mouseover event to all elements, so that their chunks are loaded on hover